File: ServiceBook/ClassLibrary1/Models/SSI.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ServiceBook.DAL.Models
{
    public class SSI
    {
        [Key]
        public int SID { get; set; }
        public int SIID { get; set; }
        [ForeignKey("SID")]
        public virtual Service Service { get; set; }
        [ForeignKey("SIID")]
        public virtual ServiceIntervention ServiceIntervention { get; set; }

    }
}

File: ServiceBook/ServiceBook/Scripts/Service/Models/SSI.js
function SSI(data) {
    this.sid = null;
    this.siid = null;
    this.service = null;
    if (data != null) {
        this.sid = data.SID;
        this.service = new Service(data.Service);
        this.siid = data.SIID;
        this.serviceIntervention = new ServiceIntervention(data.ServiceIntervention);
    }
}